src/scenes/Organizations/Detail/EditOrganization/EditOrganization.tsx

```diff
@@ -0,0 +1,60 @@
+import { useSnackbar } from 'notistack';
+import React, { FC } from 'react';
+import { Scalars, UpdateOrganizationInput } from '../../../../api';
+import { DialogState } from '../../../../components/Dialog';
+import { DialogForm } from '../../../../components/Dialog/DialogForm';
+import { SubmitError, TextField } from '../../../../components/form';
+import { OrganizationDocument } from '../OrganizationDetail.generated';
+import { useUpdateOrganizationMutation } from './EditOrganization.generated';
+
+type EditOrganizationFormProps = DialogState & { orgId: Scalars['String'] };
+
+export const EditOrganizationForm: FC<EditOrganizationFormProps> = (props) => {
+  const { orgId, ...rest } = props;
+  const [updateOrg] = useUpdateOrganizationMutation();
+  const { enqueueSnackbar } = useSnackbar();
+  const submit = async (input: UpdateOrganizationInput) => {
```

Comment:
```suggestion
  const submit = async (input: Except<UpdateOrganizationInput, 'id'>) => {
```

Reply:
Changed, it's actually removing the id from one level deeper in `input.organization.id`